recipes/diligent-core/all/conanfile.py

```diff
@@ -0,0 +1,172 @@
+                    self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version))
+        if self.settings.compiler == "Visual Studio" and "MT" in self.settings.compiler.runtime:
+            raise ConanInvalidConfiguration("Visual Studio build with MT runtime is not supported")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def package_id(self):
+        if self.settings.compiler == "Visual Studio":
+            if "MD" in self.settings.compiler.runtime:
+                self.info.settings.compiler.runtime = "MD/MDd"
+            else:
+                self.info.settings.compiler.runtime = "MT/MTd"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _patch_sources(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+    def requirements(self):
+        self.requires("opengl/system")
+
+        self.requires("libjpeg/9d")
+        self.requires("libtiff/4.3.0")
+        self.requires("zlib/1.2.11")
+        self.requires("libpng/1.6.37")
+
+        self.requires("spirv-cross/cci.20210930")
+        self.options["spirv-cross"].namespace = self.options.spirv_cross_namespace
+        self.requires("spirv-headers/1.2.198.0")
+        self.requires("spirv-tools/2021.4")
+        self.requires("vulkan-headers/1.2.198")
+        self.requires("volk/1.2.198")
+
+        if self.options.with_glslang:
+            self.requires("glslang/11.7.0")
+            self.options["glslang"].shared = True
+
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            self.requires("xorg/system")
+            if not tools.cross_building(self, skip_x64_x86=True):
+                self.requires("xkbcommon/1.3.0")        
+
+    def diligent_platform(self):
+        if self.settings.os == "Windows":
+            return "PLATFORM_WIN32"
+        elif self.settings.os == "Macos":
+            return "PLATFORM_MACOS"
+        elif self.settings.os == "Linux":
+            return "PLATFORM_LINUX"
+        elif self.settings.os == "Android":
+            return "PLATFORM_ANDROID"
+        elif self.settings.os == "iOS":
+            return "PLATFORM_IOS"
+        elif self.settings.os == "Emscripten":
+            return "PLATFORM_EMSCRIPTEN"
+        elif self.settings.os == "watchOS":
+            return "PLATFORM_TVOS"
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["DILIGENT_BUILD_SAMPLES"] = False
+        self._cmake.definitions["DILIGENT_NO_FORMAT_VALIDATION"] = True
+        self._cmake.definitions["DILIGENT_BUILD_TESTS"] = False
+        self._cmake.definitions["DILIGENT_NO_GLSLANG"] = not self.options.with_glslang
```

Comment:
I think we want to force this option to `True`.
See https://github.com/DiligentGraphics/DiligentCore/blob/4a77fda8c7de080ed5ce786b023b841a3d15fb1f/ThirdParty/CMakeLists.txt#L30
We never want the recipe to build its own glslang compiler.
So I think this option can be removed.